### PR TITLE
Responsive components

### PR DESCRIPTION
**Pull request by davidmartin84**

### CODE REVIEW

File: components/responsive/views/src/viewport-resize.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import throttle from 'lodash.throttle'
```
Comment (davidbarna) [inline components/responsive/views/src/viewport-resize.js]: We were supposed to comment on this.
This code is duplicated and should not. We could move it to @s-ui/js of to whatever other place proposed by our colleagues.

File: components/responsive/toggle/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <br />
+  <ResponsiveToggle breakpoint={575}>
+    <button>Mobile button</button>
+  </ResponsiveToggle>
+</div>)
+
+```
+
+
+> **Find full description and more examples in the [demo page](#).**
```
Comment (davidbarna) [inline components/responsive/toggle/README.md]: Add the demo page or remove this line !

File: components/responsive/views/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  <div>
+    RIGHT VIEW {current == 1 && '(active)'}
+    <button onClick={this.setView.bind(this, 0)}>Prev &lt;</button>
+  </div>
+</ResponsiveViews>
+</div>)
+```
+
+
+> **Find full description and more examples in the [demo page](#).**
```
Comment (davidbarna) [inline components/responsive/views/README.md]: Same here.

**Review by davidbarna — APPROVED** [review]

**Comment by davidmartin84** [comment]
@carlosvillu  can you check this code please?

File: components/element/breakpointToggle/src/viewport-resize.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      window.addEventListener('resize', this.update, EVENT_OPTIONS)
+    }
+  }
+
+  /**
+   * Unregister event to update viewport width and height
+   * @param {Function} listener
+   */
+  static removeListener (listener) {
+    listeners.filter(cb => cb !== listener)
```
Comment (carlosvillu) [inline components/element/breakpointToggle/src/viewport-resize.js]: I dont get this line.

**Review by carlosvillu — APPROVED** [review]

File: components/element/breakpointToggle/src/viewport-resize.js (hunk 4 above)
Comment (davidbarna) [reply components/element/breakpointToggle/src/viewport-resize.js]: It's a mistake. It should be
```js
listeners = listeners.filter(cb => cb !== listener)
```

Can you do it @davidmartin84 ?
Actually once we move it somwhere, we must test it.

Comment (davidmartin84) [reply components/element/breakpointToggle/src/viewport-resize.js]: Done!

**Comment by davidbarna** [comment]
You merged all the duplicated code of viewport-resize.js.
This MUST be solved ASAP.

**Pull request merged**